react/tic-tac-toe/src/config/api.js

```diff
@@ -0,0 +1,9 @@
+import { create } from 'apisauce';
+
+const api = create({
+  // baseURL: process.env.API_BASE_URL,
```

Comment:
use this syntax, for security reasons, we dont put our backend url in the code. you need to add API_BASE_URL to your `.env` file in the root of your project
https://medium.com/the-node-js-collection/making-your-node-js-work-everywhere-with-environment-variables-2da8cdf6e786